packages/renderer/src/lib/preferences/PreferencesRegistriesEditing.svelte

```diff
@@ -261,7 +261,7 @@ const processPasswordElement = (node: HTMLInputElement, registry: containerDeskt
 
       {#each $registriesInfos as registry}
         <!-- containerDesktopAPI.Registry row start -->
-        <div class="flex flex-col w-full border-t border-gray-900">
+        <div class="flex flex-col w-full border-t border-gray-900" aria-label="{registry.serverUrl}">
```

Comment:
For each div, that denounce the row, please use probably `row` role and `aria-label="registry.NAME?"` the property of the registry object that reference the Registry Name ( ie. "Docker Hub", "Red Hat Quay", etc.). We could then use `aria-labeledby="{registry.serverUrl}"`.

Reply:
I would not introduce another separator, instead, can we use ternary operator and use name if the property exists and url if it does not?